### ISSUE TITLE
Demo a little off

### ISSUE THREAD

**Issue by 274188A** [erossini/BlazorChartjs#15]
![image](https://user-images.githubusercontent.com/40014477/195961778-214548ec-3cd4-4b9d-a3a2-dfe9447663a4.png)

**Comment by erossini** [comment]
Oh, let me check

**Comment by erossini** [comment]
So, I think this is an issue with Chart.js. I opened a bug on [their project](https://github.com/chartjs/Chart.js/issues/10805). All the other graphs are ok except for the line graph.

**Comment by macias** [comment]
@erossini I guess your last changes within Chart.razor was about this problem? However it led only to bigger problems.

I use firefox and before when running demo I had big gap under the chart. Now, after this change I have chart overlapping the code, but in my own app the chart resizes on entire screen.

Locally I can remove those changes, but I would say until there is really rock solid solution to remove those changes from your code as well (i.e. officially so to speak).

**Comment by erossini** [comment]
I'm not sure this problem is something related to `Chart.razor` because if you open the HTML, you can see the `height` of the `chart-container` is rewritten for the `canvas` by Chart.js but only when you ask for a line chart.

I don't know if you have the same problem or not.

**Comment by erossini** [comment]
But this happens only with the line graph. For example, with Firefox

![image](https://user-images.githubusercontent.com/9497415/196745216-61160a8b-bcf7-4115-bb89-919c7b9b9d54.png)

the line graph is all over the place, but other graphs are ok.

![image](https://user-images.githubusercontent.com/9497415/196745417-8c81e371-797d-4318-9825-1e248667f63c.png)

**Comment by erossini** [comment]
I removed the `display` in the CSS

**Comment by macias** [comment]
Yes, the demo looks off, but I don't know why (so far). My point was that fix should of course improve the demo, but behave predictably in other apps as well.

**Comment by erossini** [comment]
PS: I added few new graphs
